web/js/cartographedges.js

```diff
@@ -0,0 +1,127 @@
+        var relatedPoints = '../point.json?id=' + id;
+
+        var randHue = 'rgb(' + (Math.floor(Math.random() * 256))
+                            + ',' + (Math.floor(Math.random() * 256))
+                            + ',' + (Math.floor(Math.random() * 256)) + ')';
+
+        var edgeHoverStyle =   {weight: 3,
+                                opacity: 0.95,
+                                smoothFactor: 1,
+                                attribution: 'edge'};
+        var edgeNeutralStyle = {color: randHue,
+                                weight: 1,
+                                opacity: 0.65,
+                                smoothFactor: 1,
+                                attribution: 'edge'};
+
+        $.getJSON(relatedPoints, function (data) {
+            CG.map.removeLayer(storeddotslayer);
+            CG.map.removeLayer(storedcurveslayer);
+
+            storeddots = [];
+            storedcurves = [];
+
+            var smallMarker = L.icon({
+            iconUrl: 'images/blackDot.png',
+            iconSize: [10,10]
+            });
+
+            coords = [];
+            names = [];
+
+            data[id].forEach(function (linkInfo) {
+                  var x = linkInfo.data.loc[0];
+                  var y = linkInfo.data.loc[1];
+                  marker = L.marker([x, y], {icon: smallMarker});
+                  storeddots.push(marker);
+                  coords.push([linkInfo.data.id, linkInfo.data.name, [x, y]]);
+            });
+            storeddotslayer = L.layerGroup(storeddots);
+            CG.map.addLayer(storeddotslayer);
+
+
+            var linkPairArray = [];
+            for(i=1; i < coords.length; i++) {
+                linkPairArray.push(coords[0]);
+                linkPairArray.push(coords[i]);
+            }
+
+
+            function drawCurves(){
+                json = [];
+                for (var i=0; i<linkPairArray.length-2; i+=2){
+
+                    var pointA = linkPairArray[i][2]; //source x,y
+                    var pointB = linkPairArray[i+1][2]; //dest x,y
+
+                    json.push({
+                    "id": linkPairArray[i][0] + " " + linkPairArray[i+1][0], //source + dest id
+                    "name": linkPairArray[i][1] + " -> " + linkPairArray[i+1][1], //source + dest name
+                    "data": {
+                        "coords": [
+                            pointA[0], pointA[1], pointB[0], pointB[1]
+                                  ]
+                            }
+                    });
+
+                }
+
+                var bundle = new Bundler();
+                bundle.setNodes(json);
+                bundle.buildNearestNeighborGraph();
+                bundle.MINGLE();
+
+                bundle.graph.each(function(node) { //render the graph
+                    var edges = node.unbundleEdges(1); //edges's an array of arrays that contain Graph.Node objects (which either represent an edge between two specific nodes or represent a merged edge)
```

Comment:
Single block comments are much easier to read than line-by-line comments. Could you please move what you can into larger block comments?